### PR TITLE
fetch: transfer and create transfert model

### PR DESCRIPTION
**Pull request by tsanta-niaina-MT**

### CODE REVIEW

File: src/main/java/com/hei/wallet/wallety/service/TransferService.java
Comment (sanda03) [inline src/main/java/com/hei/wallet/wallety/service/TransferService.java:1]: peux tu juste utiliser les modèles déjà existants

File: src/main/java/com/hei/wallet/wallety/endpoint/rest/model/Transfert.java
Comment (sanda03) [inline src/main/java/com/hei/wallet/wallety/endpoint/rest/model/Transfert.java:1]: Transfer sans "t"

File: src/main/java/com/hei/wallet/wallety/endpoint/rest/mapper/TransfetMapper.java
Comment (sanda03) [inline src/main/java/com/hei/wallet/wallety/endpoint/rest/mapper/TransfetMapper.java:1]: utlise les modeles existants

File: src/main/java/com/hei/wallet/wallety/endpoint/rest/model/Transfert.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Transfert {
```
Comment (RickaPrincy) [inline src/main/java/com/hei/wallet/wallety/endpoint/rest/model/Transfert.java:14]: Read the api spec... Transfer is the same transfer there ... like createdAt not created_at in there...

File: src/main/java/com/hei/wallet/wallety/endpoint/rest/controller/TransferController.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.hei.wallet.wallety.endpoint.rest.controller;
+import com.hei.wallet.wallety.model.Transfer;
+
+public class TransfertController {
```
Comment (RickaPrincy) [inline src/main/java/com/hei/wallet/wallety/endpoint/rest/controller/TransferController.java:4]: I think you can try to implement tbe crud without any features for Transfer... like getAll, getById...and save

**Review by RickaPrincy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**